README.md

```diff
@@ -5,84 +5,120 @@ Sawtooth, allowing users to exchange quantities of customized "Assets" with
-- RethinkDB's admin panel will be available at **http://localhost:9090**
+This will open a bash terminal within the shell container, with the `bin/`
+directory in the PATH, and all dependencies installed. Now to seed
+_SawbuckManager_ with all the needed Offers and Assets run:
+
+```bash
+mktadm seed --url market-rest-api:8000 --data sawbuck_app/app_data.yaml
+```
 
-Note that some dependencies may need to be installed locally, including
-Sawtooth dependencies like the`sawtooth_sdk`, as well as some `pip3` modules.
+And if you would like to schedule daily deals using _cron_, run:
+
+```bash
+mktadm schedule --daily "renew --url market-rest-api:8000 --data sawbuck_app/app_data.yaml"
+```
+
+Note that you _must_ leave this container running for it to submit new deals
+each day. If you want to simply exit the shell (this will not stop the
+container), you can simply run:
+
+```bash
+exit
+```
 
 ## Deployment
 
 Dockerfiles are also available to build images suitable for deployment, and are
-demarcated with a `-installed` tag. These will build the Protobufs and copy
-repo files at build time rather than referencing them locally. They can all be
-built individually, or built and run using `docker-compose`:
+demarcated with a `-installed` tag. These will include the source code in the
+image itself, rather than referencing what is on your local machine at runtime.
+This may be useful if you wish to distribute self-contained docker images which
+do not need the source code to run. They can be built individually using the
+`docker build` command, or all built and run together using `docker-compose`:
 
 ```bash
 docker-compose -f docker-compose-installed.yaml up
 ```
 
-Unlike the default Docker images, if the repo changes, those changes will not
-be reflected unless you rebuild them. To do that with `docker-compose`, use:
+Remember that unlike the default Docker images, if the repo changes, those
+changes will not be reflected unless the images are rebuilt. To do that with
+`docker-compose`, use:
 
 ```bash
 docker-compose -f docker-compose-installed.yaml up --build
 ```
 
-Note that this compose file only exposes the URL for the Marketplace REST API,
-not Sawtooth or RethinkDB.
+Note that this compose file only exposes the URL for _SawbuckManager_ and the
+Marketplace REST API, not Sawtooth or RethinkDB.
 
 ## Testing
 
-Integration tests can be run within docker containers using this command:
+Docker based integration tests are available. They can be built and run in one
+command:
 
 ```bash
-bin/run_integration_tests
+bin/market-integration-tests
 ```
 
 By default this command will run all tests in the `integration_tests/`
-directory. It is also possible to run a single set of tests by specifying a
+directory. It is also possible to run a single set of tests by specifying its
 sub-directory:
 
 ```bash
-bin/run_integration_tests rest_api
+bin/market-integration-tests rest_api
```

Comment:
What conventions are contributors expected to follow prior to submitting pull request, `bin/market-integration-tests` from the host environment and `bin\market` lint?

Reply:
We don't have contribution guidelines, and the testing section of this README is not meant to be a substitute for that. If our goal is to solicit contributions, we should add some scope to do a proper CI process rather than relying on people to remember to run tests.